src/main/kotlin/org/b12x/gfe/core/controller/tabstate/GfeSearchState.kt

```diff
@@ -0,0 +1,16 @@
+package org.b12x.gfe.core.controller.tabstate
+
+class GfeSearchState : TabState {
+
+    /* What tab */
+    override fun getTab(ctx: TabStateContext) = "GFE"
+
+    /* Version */
+    override fun getCurrentVersion(ctx: TabStateContext): String {
+        return "version"
```

Comment:
This whole mess may go away. I thought I needed to know which tab was requesting version (or other) data, but because all data comes through the core, I can pull it into any tab without knowing which tab requested the info.